src/main/java/org/folio/dcb/listener/CirculationCheckInEventListener.java

```diff
@@ -2,29 +2,47 @@
 
 import lombok.RequiredArgsConstructor;
 import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.repository.TransactionRepository;
 import org.folio.dcb.service.LibraryService;
 import org.folio.spring.integration.XOkapiHeaders;
 import org.folio.spring.service.SystemUserScopedExecutionService;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.messaging.MessageHeaders;
 import org.springframework.stereotype.Component;
 
+import java.util.Objects;
+
 import static org.folio.dcb.utils.TransactionHelper.getHeaderValue;
+import static org.folio.dcb.utils.TransactionHelper.parseCheckInEvent;
 
 @Log4j2
 @Component
 @RequiredArgsConstructor
 public class CirculationCheckInEventListener {
-  public final LibraryService libraryService;
+  private final LibraryService libraryService;
```

Comment:
Please rename this to lendingLibraryService and inject that bean or else we need to change this later When we implement other library roles

Reply:
Fixed